src/Functions/parseTimeDelta.cpp

```diff
@@ -168,15 +168,15 @@ namespace
                 {
                     throw Exception(
                         ErrorCodes::BAD_ARGUMENTS,
-                        "Invalid expression for function {}, don't find valid characters, str: \"{}\".",
```

Comment:
I'm also not sure where we should keep dot at the end of exception messages. In some contexts we add some punctuation after it (extra `.` or `:`), for example, see `.:` in:

```SQL
:) SELECT parseTimeDelta('xxx')
Query id: 27ceeba6-b35d-4261-a4ed-4e47364ad815
Elapsed: 0.002 sec. 
Received exception from server (version 25.7.1):
Code: 36. DB::Exception: Received from localhost:9000. DB::Exception: Invalid expression for function parseTimeDelta, find number failed, str: "xxx".: In scope SELECT parseTimeDelta('xxx'). (BAD_ARGUMENTS)
```

Reply:
I also thought about it, but let's not worry too much about it.